File: ChewieBot/ChewieBot/Commands/CommandScripts/addpointsCommand.ts
class AddpointsCommand {
    execute(username: string, data: any) {
        Console.WriteLine(`Data: ${data}`);
        var result = UserService.AddPointsForUser(data.Username, data.Points);
        var response = new CommandResponse(`${username} added ${data.Points} to ${data.Username}!`);
        return response;
    }
}

File: ChewieBot/ChewieBot/Scripting/addpointsCommand.py
import clr
clr.AddReferenceToFileAndPath("ChewieBot.Scripting.dll")
from ChewieBot.Scripting import *

def execute(username, params):
	result = UserService.AddPointsForUser(params.username, params.points)
	response = CommandResponse()
	response.message = "%s added %d points to %s" % (username, params.points, params.username)
	print response.message
	return response
    

class CommandResponse:
	message = ""